include/picongpu/ArgsParser.cpp

```diff
@@ -54,7 +54,7 @@ namespace
         /* Moving window: a new run-time parameter 'windowMovePoint' to replace
          * compile-time 'movePoint' variable
          */
-        bool isMovingWindowEnabled = vm[ "moving" ].as< bool >( );
+        bool isMovingWindowEnabled = !vm[ "moving" ].empty();
```

Comment:
Missing space between parentheses.
However, the rest of the file has a mixed style, anyway, so I would merge in order to get back to a working dev that we can use.

Reply:
Wanna follow up with a PR that regexes all `()` to `( )`? :rofl: 
... Maybe rather not. Seems like overkill.

Reply:
The problem is that some files use the old formatting style consistently, also with `< >` and other things. What we need is automatic formatting tools that cover everything, not just brackets. Like [alpaka's effort](https://github.com/ComputationalRadiationPhysics/alpaka/pull/781)

Reply:
it really doesn't matter for zero-arguments.
The idea is to group for N>=1 arguments with visual grouping ;)